### ISSUE TITLE
My Tomorrow theme mod if you like it.

### ISSUE THREAD

**Issue by Fred-Vatin** [trongthanh/vscode-boxythemekit#1]
Hi, 

Thanks for having ported this theme to VS Code.

When I used it for Sublime, I tweaked the **Tomorrow** theme because it was my favorite too.

[Here](https://gist.github.com/freMea/812dffecf0c239ef412add29b475a620) is the version for VS Code. Feel free to add it to your package as a mod, change its name, edit it, etc.

Mainly, for VS Code it only manages to change the color for: _caret_ (yellow) and _selection_ (blue).

**Comment by trongthanh** [comment]
Thanks, let me review it first.

**Comment by trongthanh** [comment]
Hi, I just reviewed you tweaks. I think it's nice but it's a bit off the original theme from Boxy ST and deserve as a new variant theme. 
My intention with this theme port is to respect original author's edit, with fixes related to coloring  compatibility with Sublime Text.
People will search for Boxy theme on VSC marketplace and expect it to look similar to ST counterpart, won't they?

I'll just leave this open in case anyone want to look for a change of the theme.

**Comment by freMea** [comment]
Yes. When the skin support in Code will be better, I'll maybe do and publish my own variant.

**Comment by trongthanh** [comment]
Just let me know if you publish your own variant from this. I'll add it to the README.
